modules/ossm-updating-istio-control-plane-with-revisionbased.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+// update/ossm-updating-openshift-service-mesh.adoc
```

Comment:
Please also check this module for any of the comments I pointed out in the previous module, and make any adjustments that also apply here (I won't point them out again here).